config/crd/bases/config.nri_balloonspolicies.yaml

```diff
@@ -268,6 +268,13 @@ spec:
                         placed on separate balloons. The default is false: prefer
                         placing containers of a pod to the same balloon(s).
                       type: boolean
+                    rdtClass:
+                      description: |-
+                        RdtClass controls RDT class of processes in the containers
+                        of a balloon. The default is empty: no RDT class is set. If
+                        defined, processes will be added to the directory
+                        corresponding the class in the resctrl file system.
+                      type: string
```

Comment:
nit: I would update the wording, since (with #483 and the other already merged related PRs) we still might end up setting some RDT class for the container (for instance based on its QoS class).

So maybe rephrase to something like this ?

```
RdtClass controls RDT class of processes in the containers
of a balloon. The default is empty: no balloon specific RDT
class is set. If defined, processes will be added to the directory
corresponding the class in the resctrl file system. Note that if
this class is omitted, the container might still get assigned to
an RDT class based on its Pod QoS class.
```